views/play2.erb

```diff
@@ -0,0 +1,7 @@
+<h1><%=@game.player2.name.capitalize%>, it's your turn!</h1>
```

Comment:
All the view files look good, calling on methods and logic from other files meaning they are only responsible for appearance